lib/rpush/client/active_record/wns/raw_notification.rb

```diff
@@ -0,0 +1,13 @@
+module Rpush
+  module Client
+    module ActiveRecord
+      module Wns
+        class RawNotification < Rpush::Client::ActiveRecord::Notification
+          validates_with Rpush::Client::ActiveModel::PayloadDataSizeValidator,
+                         limit: 5000
```

Comment:
Do you think they mean 5KB with base 10 (5 * 10^3 = 5000) and not 5KiB as in 5 * 2^10 = 5120 byte?